lib/tarantool_server.py

```diff
@@ -448,7 +448,7 @@ def seek_wait(self, msg, proc=None, name=None):
         while True:
             if os.path.exists(self.path):
                 break
-            time.sleep(0.001)
```

Comment:
Two locations with `time.sleep()` is left in `./lib/tarantool_server.py`. Why?

Reply:
It looks like they were just missed.

Reply:
Fixed.